src/app_service/service/accounts/service.nim

```diff
@@ -305,49 +186,7 @@ QtObject:
     result["KeycardPairingDataFile"] = newJString(main_constants.KEYCARDPAIRINGDATAFILE)
     result["ProcessBackedupMessages"] = newJBool(recoverAccount)
 
-  # TODO: Remove after https://github.com/status-im/status-go/issues/4977
-  proc getLoginNodeConfig(self: Service): JsonNode =
-    # To create appropriate NodeConfig for Login we set only params that maybe be set via env variables or cli flags
-    result = %*{}
-
-    # mandatory params
-    result["NetworkId"] = NETWORKS[0]{"chainId"}
-    result["DataDir"] = %* "./ethereum/mainnet"
-    result["KeyStoreDir"] = %* self.keyStoreDir.replace(main_constants.STATUSGODIR, "")
-    result["KeycardPairingDataFile"] = %* main_constants.KEYCARDPAIRINGDATAFILE
-
-    # other params
-    result["Networks"] = NETWORKS
-
-    result["UpstreamConfig"] = %* {
-      "URL": NETWORKS[0]{"rpcUrl"},
-      "Enabled": true,
-    }
-
-    result["ShhextConfig"] = %* {
-      "VerifyENSURL": NETWORKS[0]{"fallbackUrl"},
-      "VerifyTransactionURL": NETWORKS[0]{"fallbackUrl"}
-    }
-
-    result["WakuV2Config"] = %* {
-      "Port": WAKU_V2_PORT,
-      "UDPPort": WAKU_V2_PORT
-    }
-
-    result["WalletConfig"] = NODE_CONFIG["WalletConfig"]
-
-    result["TorrentConfig"] = %* {
-      "Port": TORRENT_CONFIG_PORT,
-      "DataDir": DEFAULT_TORRENT_CONFIG_DATADIR,
-      "TorrentDir": DEFAULT_TORRENT_CONFIG_TORRENTDIR
-    }
-
-    if main_constants.runtimeLogLevelSet():
-      result["RuntimeLogLevel"] = newJString(toStatusGoSupportedLogLevel(main_constants.LOG_LEVEL))
-
-    if STATUS_PORT != 0:
-      result["ListenAddr"] = newJString("0.0.0.0:" & $main_constants.STATUS_PORT)
-
+  # FIXME: remove this method, settings should be processed in status-go
```

Comment:
Should it be a new issue?

Reply:
Opened an issue: https://github.com/status-im/status-go/issues/5359